### PR TITLE
common: preventing msvc compiler warnings

### PR DESCRIPTION
**Pull request by mgrudzinska**
msvc warings regarding conversion int->float are still present

### CODE REVIEW

File: src/lib/sw_engine/tvgSwStroke.cpp [hunk 1]
```diff
@@ -40,8 +40,8 @@ static inline SwFixed SIDE_TO_ROTATE(const int32_t s)
 
 static inline void SCALE(const SwStroke& stroke, SwPoint& pt)
 {
-    pt.x *= stroke.sx;
-    pt.y *= stroke.sy;
+    pt.x = static_cast<SwCoord>(pt.x * stroke.sx);
+    pt.y = static_cast<SwCoord>(pt.y * stroke.sy);
```
Comment (mgrudzinska) [inline src/lib/sw_engine/tvgSwStroke.cpp:44]: I didn't want to cast before multiplication

File: src/loaders/svg/tvgSvgSceneBuilder.cpp [hunk 2]
```diff
@@ -80,7 +80,7 @@ static unique_ptr<LinearGradient> _applyLinearGradientProperty(SvgStyleGradient*
             stops[i].r = colorStop->r;
             stops[i].g = colorStop->g;
             stops[i].b = colorStop->b;
-            stops[i].a = (colorStop->a * opacity) / 255.0f;
+            stops[i].a = static_cast<uint8_t>((colorStop->a * opacity) / 255.0f);
```
Comment (mmaciola) [inline src/loaders/svg/tvgSvgSceneBuilder.cpp:83]: We dont need a float operation (int * int / int)

File: src/loaders/svg/tvgSvgSceneBuilder.cpp [hunk 3]
```diff
@@ -146,7 +146,7 @@ static unique_ptr<RadialGradient> _applyRadialGradientProperty(SvgStyleGradient*
...
```
Comment (mmaciola) [inline src/loaders/svg/tvgSvgSceneBuilder.cpp:149]: Same as above

**Review by mmaciola — CHANGES_REQUESTED** [review]
Please see comments. Other changes are great

File: src/loaders/svg/tvgSvgSceneBuilder.cpp (hunk 2 above)
Comment (mgrudzinska) [reply src/loaders/svg/tvgSvgSceneBuilder.cpp]: I've changed 255.0f -> 255, but I've kept casting, since opacity is int

**Review by mmaciola — APPROVED** [review]

**Pull request merged**